test/integration.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { clearStore, test, assert } from "matchstick-as/assembly/index";
+import { handleCreateSafe, handleTransferSingle } from "../src/mapping";
+import { generateCreateSafe, generateTransferSingle } from "./eventGenerators";
+import { log } from "matchstick-as/assembly/log";
+import { User } from "../generated/schema";
+
+const addressZero = '0x0000000000000000000000000000000000000000';
```

Comment:
`ethers.constants.AddressZero`

Reply:
Can't import ethers as this is technically assemblyscript